src/core/Core.js

```diff
@@ -169,6 +169,8 @@ class Uppy {
         data: file.data,
         progress: {
           percentage: 0,
+          bytesUploaded: 0,
+          bytesTotal: file.data.size || 0,
```

Comment:
So `file.data` is the result of `getItemData`, which in Dropbox/GDrive does return a `.size`